src/pages/Home/index.jsx

```diff
@@ -50,6 +55,35 @@ const Home = () => {
     }
   };
 
+  const fetchSketches = async () => {
+    try {
+      const response = await fetch(
+        `http://localhost:3000/sketches?per_page=3&page=${currentPage}`,
+      );
+      const sketchesData = await response.json();
+
+      if (sketchesData && sketchesData.sketchesUrl) {
+        setTotalPages(sketchesData.sketchesUrl.totalPages);
+        const sk = sketchesData.sketchesUrl.list.map((item) => item.imageUrl);
+        setSketches(sk);
+      }
+    } catch (error) {
+      console.error("Failed to fetch sketches:", error.message);
+    }
+  };
+
+  useEffect(() => {
+    fetchSketches();
+  }, [currentPage]);
+
+  const onPrevButtonClick = () => {
+    setCurrentPage((currentPage) => currentPage - 1);
+  };
+
+  const onNextButtonClick = () => {
+    setCurrentPage((currentPage) => currentPage + 1);
+  };
+
```

Comment:
```suggestion
  const onPrevButtonClick = () => {
    setCurrentPage((currentPage) => currentPage - 1);
  };

  const onNextButtonClick = () => {
    setCurrentPage((currentPage) => currentPage + 1);
  };

   useEffect(() => {
    fetchSketches();
  }, [currentPage]);
```
함수는 함수끼리 모아두는 것이 좋을 듯 합니다.